src/style.css

```diff
@@ -1,3 +1,103 @@
+:root {
+  --clr-primary-1: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  width: 1200px;
+}
+
+.container {
+  width: 1024px;
+}
+
+header {
+  width: 100%;
+  padding: 10px 50px;
+}
+
+.header-content-container {
+  display: flex;
+  align-items: center;
+}
+
+nav {
+  height: 60px;
+  width: 100%;
+  display: flex;
+  align-items: center;
+}
+
+ul {
```

Comment:
pls provide classnames to **all** you r elements, don't assign styles to tags, use clsses for this